alacritty_terminal/tests/ref.rs

```diff
@@ -55,6 +55,7 @@ ref_tests! {
     selective_erasure
     colored_reset
     delete_lines
+    terminal_identity_dropped
```

Comment:
We should use imperative here. So `drop_terminal_identity`.

But I don't think a reftest is the best choice here. I think unit tests would be superior.